public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -63,14 +68,24 @@ const useResourceAttributesExtensionLinks = (process: TraceProcess, datasourceTy
       return acc;
     }, {});
 
+    const spanAttributes = (spanTags ?? []).reduce<Record<string, string[]>>((acc, tag) => {
+      if (acc[tag.key]) {
+        acc[tag.key].push(tag.value);
```

Comment:
Would this cause any issues if there's the same name used in resource and span attributes? `KeyValuesTable.tsx` handles displaying only the first link 🤔